tron/api/controller.py

```diff
@@ -77,8 +77,11 @@ def handle_command(self, command):
 
     def handle_termination(self, command):
         try:
-            getattr(self.action_run, command)()
+            # Extra message is only used for killing mesos action as warning so far.
+            extra_msg = getattr(self.action_run, command)()
```

Comment:
This is cool, and in the future it would be nice to give some feedback to the user for the ssh-based killing too. (like.. did it work?)